code/game/objects/items/devices/PDA/apps.dm

```diff
@@ -129,12 +130,35 @@ var/global/list/pda_app_menus = list(
 		holomap = null
 	..()
 
+/datum/pda_app/newsreader
+	name = "Newsreader"
+	desc = "Access to the latest news from the comfort of your pocket."
+	price = 40
+	menu = PDA_APP_NEWSREADER
+	icon = "pda_news"
+	var/datum/feed_channel/viewing_channel
+	var/screen = NEWSREADER_CHANNEL_LIST
+
+/datum/pda_app/newsreader/onInstall(var/obj/item/device/pda/device)
+	..()
+
+/datum/pda_app/newsreader/Destroy()
+	..()
```

Comment:
I don't think these two supercalls are necessary, they are implicit

Reply:
yep, forgot to clean them up. (I'm at work rn, I'll do the changes tonight)